src/quetoo.h

```diff
@@ -224,6 +224,9 @@ typedef enum {
 #define QUETOO_TICK_SECONDS	(1.0 / QUETOO_TICK_RATE)
 #define QUETOO_TICK_MILLIS	(1000 / QUETOO_TICK_RATE)
 
+#define QUETOO_TO_MILLIS(t)	(t * 1000)
+#define QUETOO_TO_SECONDS(t)	(t / 1000)
+
```

Comment:
`MILLIS_TO_SECONDS`, and you probably want to divide by `1000.0` to get a fraction.